omnigibson/prims/joint_prim.py

```diff
@@ -621,7 +621,7 @@ def get_state(self, normalized=False):
         # Grab raw states
         pos = self._articulation_view.get_joint_positions(joint_indices=self.dof_indices)[0]
         vel = self._articulation_view.get_joint_velocities(joint_indices=self.dof_indices)[0]
-        effort = self._articulation_view.get_applied_joint_efforts(joint_indices=self.dof_indices)[0]
+        effort = self._articulation_view.get_measured_joint_efforts(joint_indices=self.dof_indices)[0]
```

Comment:
is this just an updated API or does this actually return something different from before?

Reply:
These methods are implemented very similarly but with one difference: 

- get_applied_joint_efforts returns the actuation forces from the physics view (the efforts set by the ``set_joint_efforts`` method)
- get_measured_joint_efforts returns the projected joint forces from the physics view (the efforts computed/measured by the physics solver of the joint forces in the DOF motion direction)

I'm not sure how much of a difference this makes in a physics simulation, but I think we still want the measured joint efforts instead of the applied. @cgokmen please double check if this is why you made the change

Reply:
Lets open a ticket about this and discuss when I'm back. I'm a bit confused about it.